sdk/storage/azblob/constants.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/internal/generated"
 )
 
+const (
+	CountToEnd = 0
```

Comment:
I don't think this is required.

Reply:
We can get rid of this and just document that 0 means "to end"

Reply:
Correct. I've removed `CountToEnd`.